src/crs/projected/eqareaalbers.jl

```diff
@@ -0,0 +1,153 @@
+
+Albers{lat₀,lat₁,lat₂,lon₀,Datum}(args...) where {lat₀,lat₁,lat₂,lon₀,Datum} =
+  Albers{lat₀,lat₁,lat₂,lon₀,Datum,Shift()}(args...)
+
+Albers(args...) = Albers{NAD83}(args...)
+
+Base.convert(
+  ::Type{Albers{lat₀,lat₁,lat₂,lon₀,Datum,Shift,M}},
+  coords::Albers{lat₀,lat₁,lat₂,lon₀,Datum,Shift}
+) where {lat₀,lat₁,lat₂,lon₀,Datum,Shift,M} = Albers{lat₀,lat₁,lat₂,lon₀,Datum,Shift,M}(coords.x, coords.y)
+
+constructor(::Type{<:Albers{lat₀,lat₁,lat₂,lon₀,Datum,Shift}}) where {lat₀,lat₁,lat₂,lon₀,Datum,Shift} =
+  Albers{lat₀,lat₁,lat₂,lon₀,Datum,Shift}
+
+lentype(::Type{<:Albers{lat₀,lat₁,lat₂,lon₀,Datum,Shift,M}}) where {lat₀,lat₁,lat₂,lon₀,Datum,Shift,M} = M
+
+==(
+  coords₁::Albers{lat₀,lat₁,lat₂,lon₀,Datum,Shift},
+  coords₂::Albers{lat₀,lat₁,lat₂,lon₀,Datum,Shift}
+) where {lat₀,lat₁,lat₂,lon₀,Datum,Shift} = coords₁.x == coords₂.x && coords₁.y == coords₂.y
+# CONVERSIONS
+# ------------
+
+# Adapted from PROJ coordinate transformation software
+# Initial PROJ 4.3 public domain code was put as Frank Warmerdam as copyright
+# holder, but he didn't mean to imply he did the work. Essentially all work was
+# done by Gerald Evenden.
+# Authors of the original algorithm: Gerald Evenden and Thomas Knudsen
+# reference code: https://github.com/OSGeo/PROJ/blob/master/src/projections/aea.cpp
+
+inbounds(::Type{<:Albers{lat₀,lat₁,lat₂,lon₀,Datum}}, λ, ϕ) where {lat₀,lat₁,lat₂,lon₀,Datum} =
+  -π ≤ λ ≤ π && deg2rad(90) ≤ ϕ ≤ deg2rad(90)
+
+function formulas(::Type{<:Albers{Datum}}, ::Type{T}) where {lat₀,lat₁,lat₂,lon₀,Datum,T}
+  # Constants
+  🌎 = ellipsoid(Datum)
+  e = T(eccentricity(🌎))
+  a = numconvert(T, majoraxis(🌎))
+
+  # Latitude origin
+  ϕ₀ = T(ustrip(deg2rad(lat₀)))
+
+  # Standard parallels
+  ϕ₁ = T(ustrip(deg2rad(lat₁)))
+  ϕ₂ = T(ustrip(deg2rad(lat₂)))
+
+  # Longitude origin
+  λ₀ = T(ustrip(deg2rad(lon₀)))
+
+  m₁ = hm(ϕ₁, e)
+  m₂ = hm(ϕ₂, e)
+  α₁ = hα(ϕ₁, e)
+  α₂ = hα(ϕ₂, e)
+  α₀ = hα(ϕ₀, e)
+  n = (m₁^2 - m₂^2) / (α₂ - α₁)
+  C = m₁^2 + n * α₁
+
+  Θ = n * (λ - λ₀)
+  ρ = a * sqrt(C - n * hα(ϕ, e)) / n
+  if ρ < 0
+    throw(DomainError("Coordinate transformation outside projection domain"))
+  end
+  ρ₀ = (a * (C - n * α₀))^0.5 / n
+  function fx(λ, ϕ)
+    ρ₀ - ρ * cos(Θ)
+  end
+
+  function fy(λ, ϕ)
+    ρ * sin(Θ)
+  end
+
+  fx, fy
+end
+
+# backward projection formulas
+
+function backward(::Type{<:Albers{Datum}}, x, y) where {lat₀,lat₁,lat₂,lon₀,Datum,T}
+  🌎 = ellipsoid(Datum)
+  e = oftype(x, eccentricity(🌎))
+  e² = oftype(x, eccentricity²(🌎))
+  ϕ₀, ϕ₁, ϕ₂, λ₀ = T.(ustrip.(deg2rad.(lat₀, lat₁, lat₂, lon₀)))
+  α₀ = hα(ϕ₀, e)
+  ρ₀ = (a * (C - n * α₀))^0.5 / n
+  α₁ = hα(ϕ₁, e)
+  α₂ = hα(ϕ₂, e)
+
+  m₁ = hm(ϕ₁, e)
+  m₂ = hm(ϕ₂, e)
+  n = (m₁^2 - m₂^2) / (α₂ - α₁)
+  C = m₁^2 + n * α₁
+
+  θ = atan2(x, ρ₀ - y)
+  ρ = sqrt(x^2 + (ρ₀ - y)^2)
+  α′ = (C - (ρ^2 * n^2) / a^2) / n
+  β′ = asin(α′ / (1 - (1 - e) / (2 * e) * log((1 - e) / (1 + e))))
+
+  λ = λ₀ + θ / n
```

Comment:
```suggestion
  λ = θ / n
```
Remove "longitude origin" from the formula.